### DISCUSSION TITLE
Groups intersection (AND) support

### DISCUSSION THREAD

**Discussion by manuel-faria** [microsoftgraph/group-membership-management#32] [Q&A]
Does this solution support Groups intersection (AND operator). Example:

There are the Security Groups "US" and "Sales". Megan wants to setup a new Teams group "Sales in US" with the people in common in both groups "US" and "Sales". Meaning that people that are only in "Sales" or only in "US" would not be added to "Sales in US".

Would that be possible?

**Comment by alrios-ms** [comment]
Hi @manuel-faria, 

Currently membership intersection is not supported. The current operation is a union of the group members.
